### ISSUE TITLE
not COFF (Check for viruses) error

### ISSUE THREAD

**Issue by akagemegane** [adbrt/cb-legacy-dev#6]
So I went to compile a simple Hello World program in DOS via DJGPP as a test and was getting this error.

![image](https://github.com/adbrt/cb-legacy-dev/assets/7905063/c3146fca-4947-44c1-a366-2e9b1649fc54)

Am I doing something wrong, or is this copy of DJGPP infected?

**Comment by C0m3b4ck** [comment]
@akagemegane no, as no modern malware would target DOS. From what I've found when searching up this issue, I found that it was related to incremental linking. Do you have that enabled by any chance? COFF means the file is corrupted, not that it has some kind of virus.

**Comment by adbrt** [comment]
This problem can happen when post-builds steps are not executed correctly, this is because DJGPP executables here have cwsdstub.exe attached into them, so you don't have to have put CWSDPMI.EXE in the application directory (this used to be a thing, and often still is, but I find this inconvenient). 

When you open project build options and check post-build steps for DJGPP, you should see this:
```
$(APPPATH)\utils\exe2coff.exe "$(PROJECT_DIR)$(TARGET_OUTPUT_DIR)$(TARGET_OUTPUT_BASENAME).exe"
cmd /C "copy /B "$(APPPATH)\utils\cwsdstub.exe" + "$(PROJECT_DIR)$(TARGET_OUTPUT_DIR)$(TARGET_OUTPUT_BASENAME)" "$(PROJECT_DIR)$(TARGET_OUTPUT_DIR)$(TARGET_OUTPUT_BASENAME).exe"
cmd /C "del "$(PROJECT_DIR)$(TARGET_OUTPUT_DIR)$(TARGET_OUTPUT_BASENAME)""
"$(APPPATH)\dosboxx\dbxlauncher.bat" "$(PROJECT_DIR)$(TARGET_OUTPUT_DIR)$(TARGET_OUTPUT_BASENAME).exe"

```
If any of these steps is not executed correctly it may leave a broken executable. This may happen because of poor handling of relative paths, special characters etc. The C::B version I used here is already pretty old and I noticed it something doesn't execute all build steps correctly. I may try to update it to a newer one.

For a quick fix, try to add a new build target (File - New - Build target), select DJGPP, save the project, reopen it and try building it again with newly created build target. In my experience it often fixed things. Sometimes even closing and reopening the project worked.
